frontend/src/metabase/routes.jsx

```diff
@@ -217,7 +217,7 @@ export const getRoutes = store => {
           </Route>
 
           <Route path="browse" component={BrowseApp}>
-            <IndexRoute component={DatabaseBrowser} />
+            <IndexRoute component={BrowseDataPage} />
             <Route path=":slug" component={SchemaBrowser} />
             <Route path=":dbId/schema/:schemaName" component={TableBrowser} />
```

Comment:
Definitely. I'm planning to do both of these steps - assigning each tab its own URL, and ensuring that the Browse data header appears on `SchemaBrowser` and `TableBrowser` - in a separate PR, since this one is already big.

Reply:
I think the URL thing is fine to do separately, but the database click removing the tabs feels like a bug that shouldn't go into master to me.